extensions/parquet/table/src/main/java/io/deephaven/parquet/table/transfer/DateTransfer.java

```diff
@@ -12,7 +12,7 @@
 
 import java.time.LocalDate;
 
-final class DateTransfer extends IntCastablePrimitiveTransfer<WritableObjectChunk<LocalDate, Values>> {
+final class DateTransfer extends IntCastablePrimitiveTransfer<ObjectChunk<LocalDate, Values>> {
```

Comment:
Some of my tests were failing with `WritableObjectChunk` for `LocalDateTimeTransfer`. So I replaced it with `ObjectChunk` and also changed the same for `DateTransfer`.

Reply:
`ObjectChunk` is correct. Please check all of the `Getting` transfers and make sure they don't use the writable interface. We're going to want to point release for that.